.eslintrc

```diff
@@ -1,13 +1,14 @@
 {
     "extends": ["eslint-config-airbnb-base" ],
```

Comment:
A very large number of rules are overridden. Perhaps there's a better base config to derive from these days. Maybe `"eslint:recommended"`?

Reply:
I agree. Years ago those rules gave a good base, but something more general would be better. `eslint:recommended` would be fine.

Reply:
Do you have plans to change this, or keep the current one?

Reply:
Oh, sorry, no I wasn't planning on making any further changes in this PR. If I ever work in this area again I might consider making an attempt, but I think this is good for now. Please, feel free to merge.